br/backup-and-restore-use-cases.md

```diff
@@ -74,26 +74,9 @@ It is recommended that you use a network disk to back up and restore data. This
 
 For the detailed usage of the `br backup` command, refer to [BR command-line description](/br/backup-and-restore-tool.md#command-line-description).
 
-1. Before executing the `br backup` command, check the value of the [`tikv_gc_life_time`](/garbage-collection-configuration.md#tikv_gc_life_time) configuration item, and adjust the value appropriately in the MySQL client to make sure that [Garbage Collection](/garbage-collection-overview.md) (GC) does not run during the backup operation.
+1. Before executing the `br backup` command, ensure that no DDL is running on the TiDB cluster.
 
-    {{< copyable "sql" >}}
-
-    ```sql
-    SELECT * FROM mysql.tidb WHERE VARIABLE_NAME = 'tikv_gc_life_time';
-    UPDATE mysql.tidb SET VARIABLE_VALUE = '720h' WHERE VARIABLE_NAME = 'tikv_gc_life_time';
-    ```
-
-2. After the backup operation, set the parameter back to the original value.
-
-    {{< copyable "sql" >}}
-
-    ```sql
-    UPDATE mysql.tidb SET VARIABLE_VALUE = '10m' WHERE VARIABLE_NAME = 'tikv_gc_life_time';
-    ```
-
-> **Note:**
->
-> Since v4.0.8, BR supports the self-adaptive GC. To avoid manually adjusting GC, register `backupTS` in `safePoint` in PD and make sure that `safePoint` does not move forward during the backup process.
```

Comment:
Is the removal intended?

Reply:
Yes. See: https://github.com/pingcap/docs/issues/4549

These are the docs for master (TiDB >5.0). The standard that the MySQL docs would use, is that this comment should appear in docs for TiDB 4.0 and later be removed. This helps improve readability, since excessive clarifications are very hard for users to read. A new user of TiDB >5.0 should not need to know the entire history of TiDB.

Reply:
LGTM